pkg/manifest/manifest.go

```diff
@@ -532,7 +532,12 @@ func UnmarshalManifest(manifestData []byte) (*Manifest, error) {
 
 func (m *Manifest) SetDefaults() {
 	if m.Image == "" && m.BundleTag != "" {
-		registry := strings.Split(m.BundleTag, ":")[0]
+		tag := m.BundleTag
```

Comment:
What do you think about using the docker library instead so that we aren't trying to replicate their logic? I think this should work, I ran it against your unit test

```go
ref, err := reference.ParseNormalizedNamed(m.BundleTag)
if err != nil {
	return errors.Wrapf(err, "invalid tag %s", m.BundleTag)
}
registry := reference.FamiliarName(ref)
```

Since ParseNormalizedName _could_ return an error when the tag is invalid, it would require `SetDefaults` changing to return an error though.

Reply:
I like it; will update.  Thank you for the head start!

Reply:
Updated!